src/hardware/sblaster.cpp

```diff
@@ -605,14 +619,29 @@ static void PlayDMATransfer(Bitu size)
 
 static void SuppressInitialDMATransfer(Bitu size)
 {
-	// Only suppress the first dword-transfer or less
-	if (size <= sizeof(uint32_t) || !sb.speaker) {
+	// Always suppress if no DMA mode is set (cannot be interpretted)
+	if (sb.dma.mode == DSP_DMA_NONE) {
 		SuppressDMATransfer(size);
-		LOG_MSG("%s: Suppressed initial %" PRIuPTR "-byte DMA transfer",
+		LOG_MSG("%s: Suppressed %" PRIuPTR
+		        " bytes of initial non-DMA-mode data.",
 		        CardType(), size);
-	} else {
+	}
+	// Otherwise suppress the first dword-sized transfer in non-16-bit DMA
+	// modes
+	else if (size <= sizeof(uint32_t) && sb.dma.mode != DSP_DMA_16 &&
+	         sb.dma.mode != DSP_DMA_16_ALIASED) {
+		LOG_MSG("%s: Suppressed %" PRIuPTR
+		        " bytes of initial %s mode data.",
```

Comment:
By my count, if format string was in a single line, it would be 86 characters-long. I find it acceptable, even if clang-format disagrees ;)

Reply:
The old highway rule.. not breaking the law if you speed 10% or less :-)
I also unwrapped the comment underneath (clocked in at 81 cols).